src/components/10-atoms/input-text/index.js

```diff
@@ -215,13 +216,19 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
   }
 
   handleInput(ev) {
+    this.cursor = ev.target.selectionStart;
```

Comment:
I suggest to reuse the existing naming: selectionStart.
(maybe alternative: targetSelectionStart)

```suggestion
    this.selectionStart = ev.target.selectionStart;
```

Reply:
Thank you, the variable can also be function scoped.